src/main/java/services/GlobalCinemaService.java

```diff
@@ -0,0 +1,67 @@
+package services;
+
+import entities.User;
+
+import java.util.Scanner;
+
+public class GlobalCinemaService {
+    public static User currentUser;
+
+    private UserService userService;
+    private BookingService bookingService;
+    private Scanner scanner;
+
+    public GlobalCinemaService() {
+        currentUser = new User();
+        scanner = new Scanner(System.in);
+
+        userService = new UserService();
+        bookingService = new BookingService(scanner);
+    }
+
+    public boolean start() {
+        System.out.println("Hello, " + currentUser.getLogin() + ", choose the option:\n" +
```

Comment:
move switch-case into another method

Reply:
It will disable exit on "9", and I will be forced to replace recurcion by while. Is it necessary?